sbom/tests/sbom.rs

```diff
@@ -1,6 +1,26 @@
+use parking_lot::Mutex;
+use sbom_walker::report;
+use std::{collections::BTreeMap, sync::Arc};
+
 #[cfg(feature = "cyclonedx-bom")]
 #[test]
 fn test_cyclonedx_v13_json() {
     let _ = sbom_walker::Sbom::try_cyclonedx_json(include_bytes!("data/cyclonedx.v1_3.json"))
         .expect("must parse");
 }
+
+#[cfg(feature = "spdx-rs")]
```

Comment:
Just for my understanding, should this be like
```suggestion
#[cfg(feature = "cyclonedx-bom")]
```
having the issue related to CycloneDX SBOM?